CHANGELOG.md

```diff
@@ -82,6 +90,26 @@
 
 [storage-mocking-link]: https://googleapis.dev/cpp/google-cloud-storage/latest/storage-mocking.html
 
+**OTHER CHANGES**:
+* feat(storage): discard handles after an error (#7088)
+* feat(storage): release download handles sooner (#7085)
+* feat(storage): support setting HTTP version (#7077)
+* fix(storage): cleanup interrupted downloads (#7064)
+* test(storage): make ScheduleForDelete() thread-safe (#7063)
```

Comment:
test-only changes are probably not interesting to customers.

Reply:
Removed.